boilerplate/app/screens/WelcomeScreen.tsx

```diff
@@ -57,7 +57,7 @@ export const WelcomeScreen: FC<WelcomeScreenProps> = observer(
             tx="welcomeScreen:readyForLaunch"
             preset="heading"
           />
-          <Text tx="welcomeScreen:exciting" preset="subheading" />
+          <Text tx="welcomeScreen:exciting" preset="subheading" style={$subheading} />
```

Comment:
These changes don't seem to render right for me on web:
<img width="1163" alt="Screenshot 2024-10-18 at 3 51 56 PM" src="https://github.com/user-attachments/assets/b66edbb9-5dfe-40de-b87e-8bbeca9a9633">